server/lib/resolvers/builders/nedb.js

```diff
@@ -16,6 +16,7 @@ function buildNeDBResolver (dataSource, compiledRequestMapping, compiledResponse
       const parsedQuery = JSONParse(queryString)
 
       if (parsedQuery.error) {
+        console.error('Error when parsing query', queryString, parsedQuery.error)
```

Comment:
Please use "log" like in other modules. There's actually a linter rule that prevents using console

Reply:
I wonder why that linter didn't complain...